src/pages/gallery/index.tsx

```diff
@@ -384,40 +384,71 @@ export default function Gallery(props) {
                                 }
                             });
 
+                            const loadMoreItems = async (startIndex, stopIndex) => {
```

Comment:
[question]
Since we have everything in local storage that too in one object, how is this helping?

Reply:
loadMoreItems loads the preview image of items by making the fetch call  not the actual items itself